core/src/main/java/hudson/console/AnnotatedLargeText.java

```diff
@@ -135,7 +135,7 @@ private ConsoleAnnotator createAnnotator(StaplerRequest req) throws IOException
             throw new IOException(e);
         }
         // start from scratch
-        return ConsoleAnnotator.initial(context==null ? null : context.getClass());
+        return ConsoleAnnotator.initial(context);
```

Comment:
This made no sense. `context` is a `T`. `initial` takes a `T`. So why were we passing a `Class<T>`?